src/main/webapp/web.xml

```diff
@@ -14,6 +14,16 @@
         <url-pattern>/</url-pattern>
     </servlet-mapping>
 
+    <servlet>
+        <servlet-name>injectData</servlet-name>
+        <servlet-class>mate.academy.controllers.InjectDataController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>injectData</servlet-name>
+        <url-pattern>/injectData</url-pattern>
```

Comment:
We shouldn't use camelSpace for urls. 
/inject-data will be better

Reply:
agree

Reply:
done)